crates/vhost-device-gpio/src/backend.rs

```diff
@@ -5,11 +5,12 @@
 //
 // SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
 
+use env_logger::Env;
```

Comment:
Should this go in the next section since it is an external crate?

Reply:
ok moved.